src/renderer/components/+workloads-cronjobs/cron-job-menu.tsx

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency